packages/core/src/utilities/validation.js

```diff
@@ -181,7 +181,7 @@ export const fallsWithinNumericalRange: FallsWithinNumericalRange = ({
 }) => {
   const parsedValue = parseFloat(value);
 
-  if (typeof value === "string" && !value.length) {
+  if (!value || (typeof value === "string" && !value.length)) {
```

Comment:
I think I'd prefer to explicitly match undefined and null here just to be clear. A value of `false` or `0` would pass - even though it should only be possible to pass a string value from the browser to the code.

Reply:
fixed